compiler/x/codegen/X86Ops.ins

```diff
@@ -2435,6 +2435,10 @@ INSTRUCTION(PADDD, paddd,
             BINARY(VEX_L128, VEX_vReg_, PREFIX_66, REX__, ESCAPE_0F__, 0xfe, 0, ModRM_RM__, Immediate_0),
             PROPERTY0(IA32OpProp_SourceRegisterInModRM),
             PROPERTY1(IA32OpProp1_XMMSource | IA32OpProp1_XMMTarget)),
+INSTRUCTION(POR, por,

```

Comment:
Why is the opcode named `POR` and not `PORRegReg` ?  It should be because its too generic otherwise.  You may add another encoding forms of this opcode (like PORRegMem) in the future and will need to distinguish them.